applications/StructuralMechanicsApplication/structural_mechanics_application.cpp

```diff
@@ -1122,7 +1123,8 @@ void KratosStructuralMechanicsApplication::Register() {
     KRATOS_REGISTER_CONSTITUTIVE_LAW("SmallStrainOrthotropicDamageSimoJu2D", mSmallStrainOrthotropicDamageSimoJu2D);
 
     // Rules of mixtures
-    KRATOS_REGISTER_CONSTITUTIVE_LAW("RuleOfMixturesLaw", mRuleOfMixturesLaw);
+    KRATOS_REGISTER_CONSTITUTIVE_LAW("ParallelRuleOfMixturesLaw2D", mParallelRuleOfMixturesLaw2D);
+    KRATOS_REGISTER_CONSTITUTIVE_LAW("ParallelRuleOfMixturesLaw3D", mParallelRuleOfMixturesLaw3D);
```

Comment:
You have changed the name to ParallelRuleOfMixturesLaw3D and ParallelRuleOfMixturesLaw2D, change the Parameters of the test to ParallelRuleOfMixturesLaw3D instead of ParallelRuleOfMixturesLaw

Reply:
@AlejandroCornejo

Reply:
I've done it!! :S

Reply:
![imagen](https://user-images.githubusercontent.com/19991680/83252967-748bfe80-a1ac-11ea-989b-011649fcd191.png)